portfolio/src/main/webapp/index.html

```diff
@@ -24,40 +24,51 @@ <h1>arrington.</h1>
     
 
     <!-- TODO: add
-        contacting arrington
+        contacting arrington (done)
         arrington's interests (done)
         arrington's details/skills (done)
         arrington's schools (done)
         arrington's companies (done)
-        arrington's blurbs
-        arrington's friend space
```

Comment:
Did you mean to remove "arrington's friend space"? It's fine just making sure it wasn't a typo.

Reply:
yep, it was on purpose!